index.html

```diff
@@ -10,21 +10,47 @@
 </head>
 
 <body>
-  <div class="awesomebooks">
-  <h1>All awesome books</h1><br>
+  <nav id="navbar">
+    <p id="navTitle">Awesome Books</p>
+    <div>
+      <ul id="nav-list">
+        <li class="list1"><a class="blue" id="listLink" href="#">List</a></li>
+        <li class="list2"><a class="link_nav" id="addLink" href="#">Add new</a></li>
+        <li class="list3"><a class="link_nav" id="contactLink" href="#">Contact</a></li>
+      </ul>
+    </div>
+  </nav>
+  <p id="date"></p>
 
-  <div id="books">
+  <div id="list-books" class="">
+    <div class="awesomebooks">
+      <h1>All awesome books</h1>
+      <div id="books"></div>
+    </div>
   </div>
-  <hr class="divisor">
 
-  <h2 class="new_book">Add a new book</h2>
-  <form action="#" class="form">
+  <h2 id="subtitle" class="new_book hide">Add a new book</h2>
+  <form id="add-books" action="#" class="form >
```

Comment:
- [ ] in the wireframe and in the project requirements, each book should have an author and a title. Kindly add the title input field to your form 👍 

![image](https://user-images.githubusercontent.com/50721479/204907181-771cd1c7-36cb-4629-af5c-9f33d859603c.png)